pegasus/sites.v3/hourofcode.com/views/promote_nav.md

```diff
@@ -1,13 +1,17 @@
-- <a href="<%= resolve_url('/resources') %>"><%= hoc_s(:nav_back_to_resources)%></a>
-- <a href="<%= resolve_url('/resources/promote#onepager') %>"><%= hoc_s(:resources_one_pager)%></a>
-- <a href="<%= resolve_url('/resources/promote#brochure') %>"><%= hoc_s(:resources_brochure)%></a>
-- <a href="<%= resolve_url('/resources/promote#videos') %>"><%= hoc_s(:resources_videos)%></a>
-- <a href="<%= resolve_url('/resources/promote#posters') %>"><%= hoc_s(:resources_posters)%></a>
-- <a href="<%= resolve_url('/resources/promote#banners') %>"><%= hoc_s(:resources_banners)%></a>
-- <a href="<%= resolve_url('/resources/promote#stickers') %>"><%= hoc_s(:resources_stickers)%></a>
-- <a href="<%= resolve_url('/resources/promote#sample-emails') %>"><%= hoc_s(:resources_emails)%></a>
-- <a href="<%= resolve_url('/resources/stats') %>"><%= hoc_s(:resources_stats)%></a>
-- <a href="<%= resolve_url('/resources/press-kit') %>"><%= hoc_s(:resources_press_kit)%></a>
+<div class="resources-navigation">
+	<div class="nav-header"><%= hoc_s(:resources_promote)%></div>
+	<div class="nav-item"><a class="nav-link" href="<%= resolve_url('/promote#onepager') %>"><%= hoc_s(:resources_one_pager)%></a></div>
```

Comment:
And as a haml file it'd probably be really clean to have a list of all the pages and just substitute they key word like "brochure" in the url and the string.